packages-exp/installations-exp/src/api/get-installations.ts

```diff
@@ -20,6 +20,8 @@ import { FirebaseInstallations } from '@firebase/installations-types-exp';
 import { _getProvider } from '@firebase/app-exp';
 
 /**
+ * Returns a Firebase Installation instance for the given app.
```

Comment:
Returns an instance of FirebaseInstallations associated with the given FirebaseApp instance.